cmd/crio/main.go

```diff
@@ -300,7 +300,7 @@ func main() {
 		cli.StringFlag{
 			Name:  "image-volumes",
 			Value: string(libkpod.ImageVolumesMkdir),
-			Usage: "image volume handling ('mkdir' or 'ignore')",
+			Usage: "image volume handling ('mkdir', 'bind', or 'ignore')",
```

Comment:
What's the behavior of bind? (totally forgotten)

Reply:
crio.conf man page needs this change also.  We should document the difference there also. 
```
       image_volumes=""
         Image volume handling ('mkdir', bind or 'ignore') (default: "mkdir")
 
When CRI-O runs an image that specifies a VOL flag, and the caller does not specify a volume to mount over this directory, it will either
- mkdir - Create a directory in local storage and allow users to create content in that directory
- bind - Bind a tmpfs on to the directory, which means if the container/pod is restarted any datat written to that path will be lost.
- ignore - Basically ignore the VOLUME flag, means that any content written to the path will be on the COW Storage associated with the container.
```

Reply:
@mrunalp Please correct my assumptions above.

Reply:
awesome, I didn't mean to block the PR for this :smile: thanks Dan!

Reply:
mkdir is where we ensure the directory in RW layer. For ignore, we just ignore the directive altogether. bind mount is an anon directory inside container directory bind mounted to volume specified directory and it goes away when container is stopped and removed.

Reply:
Ok can you clean up the crio.conf man page.